build.gradle

```diff
@@ -84,6 +84,7 @@ allprojects {
                         ).collect { it.canonicalName() } as String[])
 
                 disable 'AndroidJdkLibsChecker', // ignore Android
+                        'CanIgnoreReturnValueSuggester', // high noise on builders
```

Comment:
Latest baseline should take care of this out of the box: https://github.com/palantir/gradle-baseline/blob/4d9c322805ac8fbc1967baa529d5d1a609513fbe/gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java#L173

Reply:
https://github.com/palantir/tritium/pull/1567 to switch over to gradle-baseline provided error-prone